dependencies/models/custom.outputs

```diff
@@ -0,0 +1,4 @@
+detection_boxes
+detection_scores
+detection_classes
+num_detections
```

Comment:
These are the outputs from the SSD ResNet detection model. We can change this as needed depending on the model we end up using.